watertap/unit_models/translators/translator_asm1_adm1.py

```diff
@@ -0,0 +1,691 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Translator block representing the ADM1/ASM1 interface.
+
+Assumptions:
+     * Steady-state only
+
+Model formulated from:
+
+Copp J. and Jeppsson, U., Rosen, C., 2006.
+Towards an ASM1 - ADM1 State Variable Interface for Plant-Wide Wastewater Treatment Modeling.
+Proceedings of the Water Environment Federation, 2003, pp 498-510.
+"""
+
+# Import Pyomo libraries
+from pyomo.common.config import ConfigBlock, ConfigValue, In, Bool
+
+# Import IDAES cores
+from idaes.core import declare_process_block_class, UnitModelBlockData
+from idaes.models.unit_models.translator import TranslatorData
+from idaes.core.util.config import (
+    is_reaction_parameter_block,
+)
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.solvers import get_solver
+import idaes.logger as idaeslog
+
+from pyomo.environ import (
+    Constraint,
+    Param,
+    PositiveReals,
+    Var,
+    sqrt,
+    units as pyunits,
+    check_optimal_termination,
+    Set,
+    Expr_if,
+)
+
+from idaes.core.util.exceptions import InitializationError
+
+__author__ = "Alejandro Garciadiego"
+
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+
+@declare_process_block_class("Translator_ASM1_ADM1")
+class TranslatorDataAsm1Adm1(TranslatorData):
```

Comment:
```suggestion
class TranslatorDataASM1ADM1(TranslatorData):
```
Hate to split hairs here, but if keeping consistent with naming convention, consider that the other translator for ADM1/ASM1 uses all caps, i.e., `TranslatorDataADM1ASM1`. I'm impartial to either convention, so long as both translator class names match